src/lib/wp-cli-process.ts

```diff
@@ -122,8 +122,13 @@ export default class WpCliProcess {
 			};
 
 			const timeoutHandler = () => {
-				reject( new Error( `Request for message ${ originalMessage } timed out` ) );
+				// Kill the process on timeout
+				this.#killProcess().catch( ( error ) => {
+					console.error( 'Failed to kill process after timeout:', error );
+					Sentry.captureException( error );
```

Comment:
Not sure we necessarily need to pass this to Sentry but could be useful to debug other errors. Happy to remove this though.

Reply:
Adjusted in https://github.com/Automattic/studio/pull/741/commits